### DISCUSSION TITLE
Nod - Ubuntu

### DISCUSSION THREAD

**Discussion by DariusSporis** [CoopNetwork/GMD-blockchain#7] [Q&A] [answered]
Salutare, as avea si eu cateva intrebari:

Care dintre Ubuntu trebuie instalat? (Ubuntu Desktop sau Ubuntu Server?)
Se poate Rula Ubuntu OS pe WIndows OS sau trebuie sa renunt la Windows OS?
Trebuie sa imi creez un Stick bootabil ptr a instala Ubuntu OS sau il pot instala direct dupa ce ii dau Mount din folder?

**Comment by georgeasiminei** [comment, marked as answer]
Salutare!

1. Merg ambele variante, tine de preferinta ta daca esti obisnuit cu interfata exclusiv prin linia de comanda (pe ubuntu server), sau daca vrei si un desktop (Ubuntu Desktop). Apropos, ar trebui sa mearga orice distributie de Linux, nu doar Ubuntu.
2. Da, se poate rula Ubuntu pe windows. Sunt cateva optiuni:
Ubuntu Desktop sau Ubuntu server se poate instala in masina virtuala care ruleaza in Windows. Vezi VMWare sau VirtualBox de exemplu.
Ubuntu server poate fi rulat in in Windows 10 sau 11 folosind direct WSL (Windows Subsystem for Linux). Pasii sa activezi WSL manual si sa rulezi Ubuntu ii gasesti aici https://docs.microsoft.com/en-us/windows/wsl/install-manual
3. Daca folosesti oricare sugestie de la punctul 2 (masina virutala sau WSL), nu ai nevoie de stick bootabil, un ISO e deajuns.

**Comment by GeomaDAO** [comment]
Salut,

Contacteaza-l te rog pe @ovidiupopica pe telgram. El și-a făcut 4 noduri astăzi . Ii spui ca eu te-am trimis la el. Și te ajuta sa iei faci nod.


Cu bine ,

Arthur
